notebooks/lst_sensitivity_pyirf.ipynb

```diff
@@ -0,0 +1,1726 @@
+{
```

Comment:
<p>Use the above defined LEAKAGE2_CUT and INTENSITY_CUT</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done! Those lines are not more needed here actually. </p>